config/config.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 	LanDiscovery    bool                `json:"lan_discovery"`
 	RemoteConfig    string              `json:"remote_config,omitempty"`
 	RCLastUpdate    time.Time           `json:"rc_last_update,omitempty"`
+	// Indicates whether the virtual servers are used. True by default
+	VirtualLocation TrueField `json:"virtual_location,omitempty"`
```

Comment:
Why not `bool` as in other settings?

Reply:
The indentation is made automatically and I cannot control it, and it breaks because of the line with the comment.
I'll try to see how it looks if I mode the comment at the end of the line.

Reply:
I'll let it like this, because adding the comment on the same line will make the line too long.